### ISSUE TITLE
Attributes as sensor values

### ISSUE THREAD

**Issue by bradsour** [LordBoos/discord_game#18]
I'd like to do some filtering based on the attributes values. Like knowing when the game field is not empty so that I can flash my lights or turn them a certain color when a certain friend is playing a game. Is it possible to make it so that the attributes show up as sensors? Unfortunately at this time I don't know enough about what that would take for setup to get done.

**Comment by LordBoos** [comment]
You can use template sensors in HA configuration.
```  - platform: template
    sensors:
      sensor_name_1:
        value_template: '{{ states.sensor.discord_username_###.attributes.game}}''```

**Comment by bradsour** [comment]
I tried for hours to reference the game attribute in a filter entity only
to have to give up.  I could not figure our how to filter it out when there
was not a game populated. It worked fine if there was a game being played,
but that is not what I'm trying to accomplish. What is the value when it's
empty? I was able to get the developer tools to show it as None,  but
setting that as the value didn't work.
